tests/Examples/Search/SearchPage.cs

```diff
@@ -1,68 +1,112 @@
 using Elastic.Xunit.XunitPlumbing;
+using Elasticsearch.Net;
+using Examples.Models;
 using Nest;
 
 namespace Examples.Search
 {
 	public class SearchPage : ExampleBase
 	{
-		[U(Skip = "Example not implemented")]
+		[U]
 		public void Line7()
 		{
 			// tag::9bdd3c0d47e60c8cfafc8109f9369922[]
-			var response0 = new SearchResponse<object>();
+			var searchResponse = client.Search<Tweet>(s => s
+				.Index("twitter")
+				.QueryOnQueryString("tag:wow")
+			);
 			// end::9bdd3c0d47e60c8cfafc8109f9369922[]
 
-			response0.MatchesExample(@"GET /twitter/_search?q=tag:wow");
+			searchResponse.MatchesExample(@"GET /twitter/_search?q=tag:wow", e =>
+			{
+				e.Method = HttpMethod.POST;
+				return e;
+			});
 		}
 
-		[U(Skip = "Example not implemented")]
+		[U]
 		public void Line340()
 		{
 			// tag::be49260e1b3496c4feac38c56ebb0669[]
-			var response0 = new SearchResponse<object>();
+			var searchResponse = client.Search<Tweet>(s => s
+				.Index("twitter")
+				.QueryOnQueryString("user:kimchy")
+			);
 			// end::be49260e1b3496c4feac38c56ebb0669[]
 
-			response0.MatchesExample(@"GET /twitter/_search?q=user:kimchy");
+			searchResponse.MatchesExample(@"GET /twitter/_search?q=user:kimchy", e =>
+			{
+				e.Method = HttpMethod.POST;
```

Comment:
Does the global change for this not get applied?

Reply:
Throws an error if not included:

Examples.Search.SearchPage.Line340

```
Expected response.ApiCall.HttpMethod to be GET, but found POST.
   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.ObjectAssertions.Be(Object expected, String because, Object[] becauseArgs)
   at Examples.ResponseExtensions.MatchesExample(IResponse response, String content, Func`2 clientChanges) in C:\Source\elasticsearch-net-7.x\tests\Examples\ResponseExtensions.cs:line 42
   at Examples.Search.SearchPage.Line340() in C:\Source\elasticsearch-net-7.x\tests\Examples\Search\SearchPage.cs:line 37

```

Reply:
Added a global handle for this.